syntax = "proto3";

package csw_protobuf;

import "csw_protobuf/parameter.proto";
import "google/protobuf/timestamp.proto";
import "csw_protobuf/models.proto";

message PbEvent {
    enum PbEventType {
        ObserveEvent = 0;
        SystemEvent = 1;
    }
    string eventId = 10;
    string source = 11;
    string name = 12;
    PbUTCTime eventTime = 13;
    repeated PbParameter paramSet = 14;
    PbEventType eventType = 15;
}